csrc/python_frontend/fusion_cache.h

```diff
@@ -13,6 +13,16 @@
 
 #include <memory>
 
+#if defined(__cplusplus) && (__cplusplus >= 201703L)
+#include <filesystem>
+namespace fs = std::filesystem;
+#elif defined(__cplusplus) && (__cplusplus >= 201402L)
```

Comment:
Note: Likely not necessary? our build specifies C++17 already.
I'm fine with leaving it as-is tho